packages/streaming-image-volume-loader/src/sharedArrayBufferImageLoader.ts

```diff
@@ -0,0 +1,155 @@
+import { Enums } from '@cornerstonejs/core';
```

Comment:
You should rebase on main, this is removed

Reply:
Yeah I think this was accidentally kept during the rebase.